elixir.html.markdown

```diff
@@ -411,7 +412,8 @@ self() #=> #PID<0.27.0>
 
 * [Getting started guide](http://elixir-lang.org/getting_started/1.html) from [elixir webpage](http://elixir-lang.org)
 * [Elixir Documentation](http://elixir-lang.org/docs/master/)
-* ["Programming Elixir"](https://pragprog.com/book/elixir/programming-elixir) by Dave Thomas
+* [Programming Elixir](https://pragprog.com/book/elixir/programming-elixir) by Dave Thomas
```

Comment:
I made the (probably poor) assumption that since the title of the book is in a link, it's pretty easy to see what the title of the book is. If this is a convention that I am violating, I will gladly change it.